ssm/ssm/projects/views.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.viewsets import ModelViewSet
+from rest_framework.filters import OrderingFilter
+from rest_framework.permissions import IsAuthenticated
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.filters import OrderingFilter, SearchFilter
```

Comment:
Please change the order of imports, imports from one package should go one by one:
```
from django_filters.rest_framework import DjangoFilterBackend
from rest_framework.viewsets import ModelViewSet
from rest_framework.filters import OrderingFilter
from rest_framework.permissions import IsAuthenticated
from rest_framework.filters import OrderingFilter, SearchFilter
```